db/db_impl_compaction_flush.cc

```diff
@@ -220,6 +222,190 @@ Status DBImpl::FlushMemTablesToOutputFiles(
   return s;
 }
 
+/*
+ * Atomically flushes multiple column families.
+ *
+ * For each column family, all memtables with ID smaller than or equal to the
+ * specified ID will be flushed. Only after all column families finish flush
```

Comment:
minor: How about saying "the ID specified in `bg_flush_args`", to save the reader a little bit of effort figuring it out?